pkg/providers/pricing/suite_test.go

```diff
@@ -252,3 +260,30 @@ var _ = Describe("Pricing", func() {
 			To(ContainElements("Linux/UNIX", "Linux/UNIX (Amazon VPC)"))
 	})
 })
+
+func getPricingEstimateMetricValue(instanceType string, capacityType string, zone string) float64 {
```

Comment:
What are your thoughts on checking for region here? I understand that zone encapsulates the region, but I think we should check for region since we're adding that label in anyway.